pkg/commands/execution_context.go

```diff
@@ -301,7 +301,10 @@ func (ctx *executionContext) performShutdown() {
 				go func() {
 					defer ctx.clusterWaitGroup.Done()
 					logrus.Infof("Closing cluster %v %v", group.config.Name, curInst.id)
-					ctx.destroyCluster(curInst, false, false)
+					// TODO - what should we be doing with this error?
+					if err := ctx.destroyCluster(curInst, false, false);err != nil {
```

Comment:
I really don't know what the right thing to do with the error here is, so logged it.  If that's the right thing, I will remove the TODO comment and repush, just let me know.

Reply:
It is used only when we start cluster and if start and stop are failed, we stop retrying to create cluster for this provider. Error is logged inside, so we could have only `_ = ctx.destroyClusetr(...) ` here.